docs/guides/serving.md

```diff
@@ -90,14 +83,50 @@ URL to service endpoint: https://www.floydlabs.com/expose/mkxjJa46aJBdwP4AEdKxfU
 
 Now you can send any image file as request to this API and it will return the style transferred image.
 
+!!! important "Use the URL in the Serving tab"
+    Fixed Rest endpoint from the Serving tab on floydhub.com
+
+    ![Fixed Serving URL](../img/serving/serving_url.png)
+
+The fixed URL endpoint also provides a load-balancer behind it: this means that if you need to scale your endpoint horizontally, by adding more machines to satisty more parallel requests, you can simply launch new serving Jobs (*you need a Plan with job concurrency > 1*) and the balancer will route the traffic to the available machines.
+
+!!! important "Scale horizontally"
+	If you need to support more requests, you can add additional serving instances and scale horizontally.
+
 ```bash
-curl -o taipei_output.jpg -F "file=@./images/taipei101.jpg" https://www.floydlabs.com/expose/mkxjJa46aJBdwP4AEdKxfU
+curl -o taipei_output.jpg -F "file=@./images/taipei101.jpg" https://www.floydlabs.com/serve/narenst/projects/fast-style-transfer
 ```
 
 ![Muse](../img/taipei_muse.jpg)
 
 You will see the default style ([la_muse](https://github.com/floydhub/fast-style-transfer/blob/master/examples/style/la_muse.jpg)) is applied to the input image.
 
+### Build an interactive website upon Serving
+
+Serving is not limited to single REST endpoint. You can build complex interactive website at its top. Here's an example.
```

Comment:
I think more details around how to do this, like we support basic Flask templating. Also note if there are any limits. And possibility, would be good to show some snippets for common UI components. This last bit might be out of scope for these changes, but I think they would help/